tests/thelp.nim

```diff
@@ -1,13 +1,32 @@
-import std/[unittest, strutils]
-import moepkg/[unicodeext, gapbuffer]
+import std/[unittest, strutils, importutils]
+import moepkg/[unicodeext, gapbuffer, editorstatus, bufferstatus]
 
 import moepkg/helputils {.all.}
+import moepkg/help {.all.}
+import moepkg/commandline {.all.}
 
-suite "Help":
+suite "initHlepModeBuffer":
   test "initHlepModeBuffer":
```

Comment:
@fox0430

Please fix the typo `Hlep` (`e` and `l` exchanged) in lines 8 and 9.

```suggestion
suite "initHelpModeBuffer":
  test "initHelpModeBuffer":
```